### DISCUSSION TITLE
How to obtain the source quality(best quality)?

### DISCUSSION THREAD

**Discussion by orloko** [streamlink/streamlink#3700] [Q&A]
by "Source Quality" I mean the best quality that you can watch on Youtube.

is it possible? footage displayed on VLC using `$ streamlink [youtube] best` looks much more blurrier and also introduce "pixelation" on dark areas during movements or camera changes.

Image for reference, top is from youtube, bottom from VLC
https://i.imgur.com/o6NOVRo.png

Skin/Hair/Beard on Youtube looks much better, footage in VLC looks worse in general.

**Comment by jpeaglesandkatz** [comment]
Seems it can only handle max 1080P which for me makes this whole app null and void.

**Comment by WolfganP** [comment]
You may need to check the debug output of streamlink + media info from VLC (and video filters used by default) + nerd stats from YT to properly compare what are you viewing in each window...

**Comment by gravyboat** [comment]
Most higher quality youtube options are only available via DASH which we don't support currently. There's a huge discussion about it here: https://github.com/streamlink/streamlink/issues/2936
